packages/@ngtools/logger/src/logger.ts

```diff
@@ -29,7 +29,7 @@ export class Logger extends Observable<LogEntry> {
   protected _metadata: LoggerMetadata;
```

Comment:
Take a look at the logger implemented in the devkit. DevKit is already strickNullCheck so I had to do this work already. I don't use any `!`, instead using the proper variable and checking if it's set or not.